articles/article-025-we-all-hate-bugs.md

```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs are something nobody wants. They creep into your house or your code without notice. There they eat your resources and clog your sink. 
```

Comment:
I would not refer to code in this sentence. The concept of a software bug is introduce only later.